crates/viewer/re_viewer/src/app.rs

```diff
@@ -1589,6 +1593,97 @@ impl App {
             .success(format!("Copied {url:?} to clipboard"));
     }
 
+    fn copy_entity_hierarchy_to_clipboard(
```

Comment:
let's not dump all of this into app.rs and split this up instead. The entity tree itself could own `format_entity_tree` which can also be unit tested there then

Reply:
this then can also be part of the dbg output of the entity tree